src/Parser.js

```diff
@@ -506,6 +514,18 @@ Parser.prototype.parseFunction = function(pos, mode) {
     }
 };
 
+/**
+ * Call a function handler with a suitable context and arguments.
+ */
+Parser.prototype.callFunction = function(name, data, args, positions) {
```

Comment:
We do this with parseArguments too where we do the lookup before passing in "funcData".